client/src/app.jsx

```diff
@@ -13,7 +12,8 @@ class App extends React.Component {
     this.state = {
       productID: 14450,
       styleID: 73180,
-      averageRating: 3.5
+      // Do we need this?
```

Comment:
Kimberly and I both need this information for our modules. Currently, we are writing separate GET requests in our components. When we are in the refactoring phase, we will have one GET request in App and pass this information to our components.